dev-python/markdown/markdown-3.1.1.ebuild

```diff
@@ -3,7 +3,9 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python2_7 python3_{6,7} pypy3 )
+PYTHON_COMPAT=( python3_{6,7,8} pypy3 )
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
```

Comment:
It's not "broken" though is it? This change requires a revbump also.

Reply:
Sorry, I wasn't very specific on what is broken

The docs flag is broken `esetup.py build_docs` is not a valid command and fails.

Also, when emerging it, it outputs a message telling me this `DISTUTILS_USE_SETUPTOOLS=rdepend` should be in the ebuild.